lib/pbench/server/api/resources/users_api.py

```diff
@@ -474,6 +450,41 @@ def put(self, username):
                 403, message="Authentication token does not belong to the current user"
             )
 
+        # Check if the user payload contain fields that are either non-updatabale or
```

Comment:
"updateabale" probably is not the word you were looking for. ;-)

I think this more appropriately placed and descriptive comment also supersedes the comment below at line 475, which is too late to be really useful and probably should be removed... maybe along with the whole `KeyValue` exception handler, which I believe we shouldn't be able to reach if we've already diagnosed and aborted on non-existent payload fields here.